### ISSUE TITLE
Cookies are shared between localhost servers started on different ports

### ISSUE THREAD

**Issue by Leksat** [AmazeeLabs/silverback#62]
I found this:
- https://www.drupal.org/project/drupal/issues/2933569 (just a summary)
- https://stackoverflow.com/questions/1612177/are-http-cookies-port-specific (says that cookies are only domain specific and not port specific)

In general, it looks like browsers do not support port specific cookies.

It means that:
- it's not really possible to run two Silverback projects locally at the same time
- `silverback dev` and `silverback test` commands (https://github.com/AmazeeLabs/silverback/tree/master/bin) may work unreliable, and developers can lose time trying to figure out what's wrong

**Comment by dan2k3k4** [comment]
Perhaps managing `/etc/hosts` file to add something like:
`localhost http://<silverback-project>` would get around this issue?

**Comment by pmelab** [comment]
True. I'm running cypress tests in Chrome Canary. That's why I didn't encounter this yet. My initial reason was that I didn't want policy-enforced plugins to cause errors while testing (looking at you, LastPass).

Using a separate browser helps, we should add that to the docs.

**Comment by pmelab** [comment]
But doesn't cypress open a clean managed session by now?

**Comment by Leksat** [comment]
Yeah, probably not an issue for Cypress tests.

> Using a separate browser helps, we should add that to the docs.

Where to add this? Create a new "Gotchas" section?
